<ui version="4.0" >
 <class>mediaInfoForm</class>
 <widget class="QWidget" name="mediaInfoForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QPushButton" name="pbEdit" >
       <property name="text" >
        <string>Edit</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >
         <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPurge" >
       <property name="text" >
        <string>Purge</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >
         <normaloff>:/images/purge.png</normaloff>:/images/purge.png</iconset>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >
         <normaloff>:/images/purge.png</normaloff>:/images/purge.png</iconset>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPrune" >
       <property name="text" >
        <string>Prune</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >
         <normaloff>:/images/edit-cut.png</normaloff>:/images/edit-cut.png</iconset>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbLoad" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Load</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >
         <normaloff>:/images/intern.png</normaloff>:/images/intern.png</iconset>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbUnload" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Unload</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >
         <normaloff>:/images/extern.png</normaloff>:/images/extern.png</iconset>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>251</width>
         <height>241</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>251</width>
         <height>243</height>
        </size>
       </property>
       <property name="title" >
        <string>Information</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_17" >
          <property name="text" >
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_VolumeName" >
          <property name="text" >
           <string>Vol0001</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Pool:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="label_Pool" >
          <property name="text" >
           <string>Default</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Online:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="label_Online" >
          <property name="text" >
           <string/>
          </property>
          <property name="pixmap" >
           <pixmap resource="../main.qrc" >:/images/inflag0.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Enabled:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="label_Enabled" >
          <property name="text" >
           <string>yes</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLabel" name="label_Location" >
          <property name="text" >
           <string>Vault</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QLabel" name="label_VolStatus" >
          <property name="text" >
           <string>Append</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="label_33" >
          <property name="text" >
           <string>Media Type:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QLabel" name="label_MediaType" >
          <property name="text" >
           <string>File</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0" >
         <widget class="QLabel" name="label_35" >
          <property name="text" >
           <string>Recycle Pool:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" >
         <widget class="QLabel" name="label_RecyclePool" >
          <property name="text" >
           <string>Scratch</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>261</width>
         <height>241</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>261</width>
         <height>241</height>
        </size>
       </property>
       <property name="title" >
        <string>Statistics</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Vol Bytes:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_VolBytes" >
          <property name="text" >
           <string>19.8 MB</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_11" >
          <property name="text" >
           <string>Vol Mounts:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="label_VolMounts" >
          <property name="text" >
           <string>10</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_12" >
          <property name="text" >
           <string>Recycle count:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="label_RecycleCount" >
          <property name="text" >
           <string>5</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_13" >
          <property name="text" >
           <string>Read time:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="label_VolReadTime" >
          <property name="text" >
           <string>10 mins</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_14" >
          <property name="text" >
           <string>Write time:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLabel" name="label_VolWriteTime" >
          <property name="text" >
           <string>20 mins</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="label_15" >
          <property name="text" >
           <string>Errors:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QLabel" name="label_VolErrors" >
          <property name="text" >
           <string>0</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="label_31" >
          <property name="text" >
           <string>Last Written:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QLabel" name="label_LastWritten" >
          <property name="text" >
           <string>2009-07-05 12:23:00</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0" >
         <widget class="QLabel" name="label_16" >
          <property name="text" >
           <string>First Written:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" >
         <widget class="QLabel" name="label_FirstWritten" >
          <property name="text" >
           <string>2009-06-05 10:00:00</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>241</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>200</width>
         <height>241</height>
        </size>
       </property>
       <property name="title" >
        <string>Limits</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Use duration:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_VolUseDuration" >
          <property name="text" >
           <string>0</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_10" >
          <property name="text" >
           <string>Max jobs:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="label_MaxVolJobs" >
          <property name="text" >
           <string>0</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_37" >
          <property name="text" >
           <string>Max files:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="label_MaxVolFiles" >
          <property name="text" >
           <string>0</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_38" >
          <property name="text" >
           <string>Max bytes:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="label_MaxVolBytes" >
          <property name="text" >
           <string>0</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_41" >
          <property name="text" >
           <string>Recycle:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QCheckBox" name="chkbox_Recycle" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Retention:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QLabel" name="label_VolRetention" >
          <property name="text" >
           <string>365 days</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Expire:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QLabel" name="label_Expire" >
          <property name="text" >
           <string>2010-08-03 23:10:03</string>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Jobs</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QTableWidget" name="tableJob" >
        <property name="sortingEnabled" >
          <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>JobId</string>
         </property>
        </column>
        <property name="sortingEnabled" >
          <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Name</string>
         </property>
        </column>
        <property name="sortingEnabled" >
          <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Start Time</string>
         </property>
        </column>
        <property name="sortingEnabled" >
          <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Type</string>
         </property>
        </column>
        <property name="sortingEnabled" >
          <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Level</string>
         </property>
        </column>
        <property name="sortingEnabled" >
          <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Files</string>
         </property>
        </column>
        <property name="sortingEnabled" >
          <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Bytes</string>
         </property>
        </column>
        <property name="sortingEnabled" >
          <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Status</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
